keras_cv/metrics/coco/recall.py

```diff
@@ -22,13 +22,16 @@
 from keras_cv.metrics.coco import utils
 
 
-class _COCORecall(keras.metrics.Metric):
-    """_COCORecall computes the COCO recall metric.
+class _BoxRecall(keras.metrics.Metric):
+    """_BoxRecall computes recall based on varying true positive IoU thresholds.
 
-    A usage guide is available on keras.io:
-    [Using KerasCV COCO metrics](https://keras.io/guides/keras_cv/coco_metrics/).
-    Full implementation details are available in the
-    [KerasCV COCO metrics whitepaper](https://arxiv.org/abs/2207.12120).
+    _BoxRecall is analagous to traditional Recall.  The primary distinction is
+    that when operating in the problem domain of object detection there exists
+    ambiguity in what is considered a true positive.  The _BoxRecall metric
+    works by using the Intersection over Union (IoU) metric to determine whether
+    or not a detection is a true positive or a false positive.  An average
```

Comment:
Please specify the criteria more explicitly

Reply:
Thanks - good point this was ambiguous.  Addressed